index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+
+      <div id="title">
+        Dr. Norman Borlaug
+        <div id="tribute-info">The man who saved a billion lives</div>
+      </div>
```

Comment:
Using a <header> tag with <h1> for title can improve semantics and UX. You can read more about this [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)